concourse/tasks/gcloud-insert-image.yaml

```diff
@@ -0,0 +1,22 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+inputs:
+- name: compute-image-tools
+
+params:
+  TOPIC: "projects/artifact-releaser-prod/topics/gcp-guest-image-release-prod"
+
+run:
+  path: sh
+  args:
+  - -exc
+  - |
+    wf=$(sed 's/\"/\\"/g' ./compute-image-tools/daisy_workflows/build-publish/((wf)) | tr -d '\n')
+    gcloud pubsub topics publish $TOPIC --message "{\"type\": \"insertImage\", \"request\": {\"image_name\": \"((image_name))\", \"gcs_image_path\": \"((gcs_image_path))\", \"image_publish_template\": \"${wf}\", \"source_version\": \"((source_version))\", \"publish_version\": \"((publish_version))\", \"release_notes\": \"((release_notes))\"}}"
```

Comment:
From what I can tell, the purpose of this is to escape a text file into a json string?

I'm concerned about two things:
1. The readability and maintainability of the `sed` approach -- for me, at least, it's really cryptic.
2. The correctness of the escaping.

I'd suggest two alternative approaches: 
1. Write a small container that becomes an interface for communicating with artifact releaser
2. If that's an unjustifiably-large task, use a proper JSON library; I would assume that jq is smart enough to handle this.